### DISCUSSION TITLE
file count: undefined

### DISCUSSION THREAD

**Discussion by Viiprogrammer** [verdaccio/verdaccio#3737] [General]
![image](https://user-images.githubusercontent.com/17622604/232312101-57957512-9788-4f77-82cd-5c4c9283d9f3.png)

Problem not depends on package content / package-json (same with `npm init -y && npm publish`)

`verdaccio` installed via npm: `npm i verdaccio -g` with zero config run from terminal.

`OS`: `Ubuntu 22.04`
`NPM`: `8.15.0`
`Node`: `16.17.0`
`verdaccio`: `5.23.2`

**Comment by juanpicado** [comment]
Try again with latest.

**Reply by Viiprogrammer** [reply]
Works fine
![image](https://user-images.githubusercontent.com/17622604/234028462-143f1437-848c-48dc-93b0-29dc7ce2ae79.png)
